src/Windows/Avalonia.Direct2D1/Media/FormattedTextImpl.cs

```diff
@@ -64,9 +64,27 @@ public class FormattedTextImpl : IFormattedTextImpl
 
         public DWrite.TextLayout TextLayout { get; }
 
+        private bool _disposed;
+
         public void Dispose()
         {
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
This class doesn't manage any native handles, therefore it shouldn't have finalizer and shouldn't use two-stage Dispose pattern.